lib/workspace.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "user"
+require_relative "channel"
+
+module Slack
+  class Workspace
+    attr_reader :users, :channels
+    attr_accessor :selected
```

Comment:
I think an `attr_reader` might be the right call for `selected` as well, since it's only changing from within this class.